### ISSUE TITLE
Android compatibility

### ISSUE THREAD

**Issue by Trass3r** [tonihele/OpenKeeper#218]
JME itself has good Android support: http://docs.jmonkeyengine.org/jme3/android.html
Are there any long-term plans to work towards Android compatibility?

**Comment by tonihele** [comment]
No, not at the moment. Primary focus is on Win/Linux/Mac, well, PC. Enhance the experience and add some spark to it, the modern goodies. And that is kinda working away from Android. But not ruling it out of course.

I've no objection if somebody picks up this feature. As long as it doesn't hinder the primary platform development. I'm still on my Samsung S2, so no help from me :)

**Comment by Trass3r** [comment]
I see. As far as I know one would have to get away from awt/swing. Not sure if there's more regarding the core game, i.e. excluding tools etc.

**Comment by tonihele** [comment]
We're not using awt/swing, as far as I know. Changes are OpenKeeper would start now with Android, but probably too hard to use, if it doesn't crash before.

**Comment by Trass3r** [comment]
When searching the repo it seems to use JFrame in the main and a bunch of awt Events, Points, Graphics etc.

**Comment by tonihele** [comment]
Ah oh yes those. If Points need to replaced, that would be really simple. Swing is for the setup, but that is one thing I don't know how Android should do it really (we ask for the game directory).

**Comment by ufdada** [comment]
Well technically we could do it ingame, but on the other hand i think it's not really pretty to choose an installation-folder in android. We should think about a smarter way on other plattforms than windows anyway.

**Comment by Trass3r** [comment]
https://github.com/search?q=repo%3Atonihele%2FOpenKeeper%20awt&type=code
https://github.com/tonihele/OpenKeeper/issues/304#issuecomment-352272004
